spec/filters/cidr_spec.rb

```diff
@@ -102,5 +104,52 @@
        insist { subject.get("tags") }.nil?
     end
   end
+  
+  describe "Load network list from a file" do
 
+    let(:network_path) {File.join(File.dirname(__FILE__), "..","files","network")}
+    let(:config) do
+      "filter { cidr { network_path => \"#{network_path}\" address => \"%{clientip}\" add_tag => \[\"matched\"] }}"
+    end
+
+    sample("clientip" => "192.168.1.1") do
+      insist { subject.get("tags") }.include?("matched")
+    end
```

Comment:
How about also testing that an IP address for the second range in the test file (200.17.160.0/24) matches so that we test that the whole file is loaded?